internal/connectors/plugins/public/column/client/timeline.go

```diff
@@ -0,0 +1,57 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+// Timeline allows the client to navigate the backlog and decide whether to fetch
+// historical or recently added data
+type Timeline struct {
+	LatestID      string `json:"latest_id"`
+	BacklogCursor string `json:"backlog_cursor"`
+}
+
+func (t Timeline) IsCaughtUp() bool {
```

Comment:
```suggestion
func (t *Timeline) IsCaughtUp() bool {
```
I'm new to Go, but I think pointers here are somewhat better? I don't think it matters much at all here, but hey.

Reply:
I have seen some horror debugging happen because someone decided to transform values in a pointered struct somewhere in a subroutine of a subroutine that wasn't obvious, so it can be a good idea to pass by reference and give the caller the full control of how to use the original object & the new transformed one.

But I agree if you have a function like `UpdateTimeline` where it is pretty obvious what the subroutine will do, using a pointer works nicely too 👍.